portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -43,13 +43,53 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132191 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection",
+			formPageNumber = "0");
+
+		FormViewBuilder.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Numeric",
+			formPageNumber = "0");
+
+		Click(
+			key_name = "Single Selection",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		Click(locator1 = "Icon#VERTICAL_ELLIPSIS_BUTTON");
+
+		Click(
+			key_selectOption = "Delete",
+			locator1 = "FormViewBuilder#SELECT_FIELD_OPTIONS_LIST");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		AssertElementNotPresent(
+			key_name = "Single Selection",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
```

Comment:
This looks good, but it would be even better if created inside a new macro, so we would be able to reuse it in other tests - I’ve checked and we don’t have one already created for this. Once we will have “can be deleted” for other fields, it will be really helpful :)

May I ask you to to create a macro that fills this scope?

A few notes:

- I believe the best place to create it would be on `FormFields.macro` file
- It would be great if this macro could receive the **field label** and **field reference** as parameters

Thank you!

Note: You can find [here](https://github.com/liferay-forms/liferay-portal/pull/265/commits/d0ce2b473a42aeebd63a3f71700e4c06ef6ccb3f) how to retrieve the field reference variable, as per @evanilson-santana solution.